### ISSUE TITLE
Support signing up to quicksy.im service

### ISSUE THREAD

**Issue by pravi** [dino/dino#1171]
Quicksy.im makes signing up and using xmpp easier by offering sign up via text message sent to phone. So users don't have to remember xmpp server address, username or password (a random password is set by default). But quicksy is currently available only on android.

In Quicksy, user enters a phone number, this will send a request to quicksy.im service via its api, which will send an otp as text message. Upon entering this otp, the app verifies the otp matches the one sent by server and sign up is completed.

Dino already works with quicksy.im service by entering username and password (+xxxxxxxxxxx@quicksy.im and random password set by quicksy app).

It would be nice to offer this sign up option directly in Dino so people with android can also use Quicksy (laptops or phones like PinePhone or Purism Librem 5).

If this can't be supported directly, would it be possible to offer this as a build time configuration option ?

Basically Quicksy makes joining xmpp very easy like how people are used to with WhatsApp, Telegram, Signal etc. But unlike those other apps, it is just another xmpp service so interoperable with every other xmpp services avoiding a lock-in. This gives an option for those who don't like Quicksy service or don't want to share phone number to stay out of Quicksy, but still talk to Quicksy users using any other xmpp service.
